src/less/mixins/utility/utility-mixins.less

```diff
@@ -22,3 +22,8 @@
     text-overflow: ellipsis;
     white-space: nowrap;
 }
+
+.customProperty(@property, @propertyValue){
```

Comment:
Nice mixin. Do you want to try using it in this PR?

Reply:
Yep, already applied it to Breadcrumb